invenio_rdm_records/contrib/meeting/sort.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+MEETING_SORT_OPTIONS = {
+    "conference-desc": {
+        "title": ("Conference [Newest]"),
+        "fields": [
+            "-custom_fields.meeting:meeting.session",
```

Comment:
I think we will have to display `Conference session` or `Conference talk` in the end, since it is quite specific, and having only `Conference` might be confusing

Reply:
Implementing the change, but it might need  to change the size of the sort

![image](https://user-images.githubusercontent.com/6756943/227501801-c36837a8-a91b-4b7a-b66a-90bb20e59caf.png)

Reply:
Added details in the zenodo-rdm PR